features/Dockefile.buildimage

```diff
@@ -0,0 +1,25 @@
+FROM fedora:30 as build-tools
```

Comment:
Any particular reason of using fedora:30 instead fedora:31? also the `as build-tools` I think it is not needed as it is not a multistage build (there is no FROM build-tools later)

Reply:
None, a part the fact that I "took inspiration from" a Dockerfile built like that.
Fixing.

Reply:
Done